.github/workflows/php-tests-wp-latest-php-7-4.yml

```diff
@@ -66,9 +66,8 @@ jobs:
           php-version: '7.4'
       - name: Get Composer Cache Directory
         id: composer-cache
-        run: |
-          echo "::set-output name=dir::$(composer config cache-files-dir)"
-      - uses: actions/cache@v2
+       run: echo "dir=$(composer config cache-files-dir)" >> $GITHUB_OUTPUT
```

Comment:
Whoops! I didn't spot this before merging, but it's broken the YML leading to [failures running the workflow](https://github.com/google/site-kit-wp/actions/runs/3523314627):

```suggestion
        run: echo "dir=$(composer config cache-files-dir)" >> $GITHUB_OUTPUT
```

![image](https://user-images.githubusercontent.com/18395600/203318091-0d118e14-1682-44bb-8972-2f2961413fd9.png)